lib/libutee/include/pta_invoke_tests.h

```diff
@@ -73,5 +73,23 @@
  */
 #define PTA_INVOKE_TESTS_CMD_LOCKDEP		8
 
+#define PTA_INVOKE_TESTS_MODE_ECB		0
```

Comment:
1. Comment needed: the values should match those defined in the user-space test TA (optee_test/ta/aes_perf/include/ta_aes_perf.h)
2. I suggest adding `_AES_`: `PTA_INVOKE_TESTS_AES_MODE_ECB` etc.

Reply:
I'll fix